.github/workflows/zowe-cli-plugin.yml

```diff
@@ -12,7 +12,7 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                node-version: [16.x, 18.x]
+                node-version: [18.x]
```

Comment:
Should we add 20.x to the list since that's also currently a supported LTS version?